src/openrct2/game.h

```diff
@@ -150,6 +150,7 @@ extern bool gInUpdateCode;
 extern bool gInMapInitCode;
 extern sint32 gGameCommandNestLevel;
 extern bool gGameCommandIsNetworked;
+extern char gCurrentLoadedPath[260];
```

Comment:
Or just use _MAX_PATH like I originally had? I get the idea of wanting this part consistant, but it feels like the PR is being held because people either don't understand what this does (because of confusion with gScenarioSavePath, when this acts differently), or because this is 260 (as gScenarioSavePath is also a raw number in the header). I know I'm probably going to sound like an asshole with this comment, but it's beginning to really bother me that the marker was put in broken and has been broken for a month.  The marker has only one job and it's not being done. 

Or can I submit a pr that at least hides the marker until this is merged?